core/server/master/src/main/java/alluxio/master/meta/AlluxioMasterRestServiceHandler.java

```diff
@@ -1121,19 +1119,6 @@ public Response getWebUIMetrics() {
     }, ServerConfiguration.global());
   }
 
-  private Comparator<String> getTierAliasComparator() {
```

Comment:
just a sanity check, is this not used anywhere?

Reply:
It was grey according to IntelliJ. Consider that this is a private function, I'd trust if it's not called in this file, it's not used?